lightning/src/ln/functional_tests.rs

```diff
@@ -25,6 +25,7 @@ use ln::{chan_utils, onion_utils};
 use ln::chan_utils::HTLC_SUCCESS_TX_WEIGHT;
 use routing::network_graph::{NetworkUpdate, RoutingFees};
 use routing::router::{Route, RouteHop, RouteHint, RouteHintHop, get_route, get_keysend_route};
+use routing::scorer::DefaultScorer;
```

Comment:
Can we use either a `TestScorer` in tests or a `DefaultScorer` with a score of 0? I find it a bit...awkward? to change all of our routing parameters in all our tests, even if they still pass it kinda makes me nervous.

Reply:
Sure thing. I removed `TestScorer` and renamed `DefaultScorer` to `Scorer`, using it with 0 in all tests excepted for the newly added one.